src/web/components/SharingPermission/SharingPermissionsTable.tsx

```diff
@@ -69,11 +70,16 @@ export function SharingPermissionsTable({
           open={openConfirmation}
           onOpenChange={setOpenConfirmation}
         >
-          <ul className='dot-list'>
-            {selectedParticipantList.map((participant) => (
-              <li key={participant.id}>{participant.name}</li>
-            ))}
-          </ul>
+          <div className='dialog-body-section'>
+            <ul className='dot-list'>
+              {selectedParticipantList.map((participant) => (
+                <li key={participant.id}>{participant.name}</li>
+              ))}
+            </ul>
+            <p>
+              Note: Sharing will continue with participants that are shared via &quot;Auto&quot;.
```

Comment:
I know this isn't the exact copy in the ticket, but since this message will currently always show up, rather than just when trying to delete 'Auto' sharing participants, I thought I would adjust it slightly.